src/db/init.go

```diff
@@ -44,8 +45,13 @@ func InjectDBProvider(name string, provider DatabaseProvider) {
 }
 
 func init() {
-	InjectDBProvider("postgres", ProvidePostgresDB)
-	InjectDBProvider("clickhouse", ProvideClickhouseDB)
+	if config.UseFileDB {
```

Comment:
as per golang's package initialization spec and setup in this repo, its guaranteed that config package will be initialized before db